src/utils/viewer/viewer.cpp

```diff
@@ -142,6 +154,8 @@ Mouse Functions:
     (With 'enable-physics') Click a surface to instance a random primitive object at that location.
   SHIFT-RIGHT:
     Click a mesh to highlight it.
+  WHEEL:
+    Modify orthographic camera zoom/perspective camera FOV (+SHIFT for fine grained control)
```

Comment:
Does wheel support pinch scrolling laptops? I don't mind plugging in a mouse for that feature, but it would be nice to add one last key binding for that.

Reply:
It does, it always did.